Sources/MapboxMaps/Gestures/GestureHandlers/PinchGestureHandler.swift

```diff
@@ -34,12 +37,20 @@ internal class PinchGestureHandler: GestureHandler {
              TODO: Handle a concurrent rotate gesture here.
              Prioritize the correct gesture by comparing the velocity of competing gestures.
              */
-
+            previousPinchCenterPoint = pinchCenterPoint
         } else if pinchGestureRecognizer.state == .changed {
-
+            
```

Comment:
nit: whitespace

Reply:
Good catch. Linter wasn't set up correctly and I miss several of these.